From 85ca82b21c5e2ea7cc28b079acc224b534d8d357 Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Thu, 2 Jul 2015 13:29:04 +0300
Subject: [PATCH 13/66] configs: Update armv8 defconfigs to use locally
 installed compiler

	By default, the path is set to an illegal compiler, to make the user
	change this to his own compiler.

Change-Id: Ic4be67bbeee531a3cf2558391a23983a9fa1d286
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/20986
Reviewed-by: Yehuda Yitschak <yehuday@marvell.com>
---
 configs/mvebu_armv8_le_defconfig      | 6 ++++++
 configs/mvebu_armv8_tiny_le_defconfig | 6 ++++++
 2 files changed, 12 insertions(+)

diff --git a/configs/mvebu_armv8_le_defconfig b/configs/mvebu_armv8_le_defconfig
index 7d033b5..48eafc4 100644
--- a/configs/mvebu_armv8_le_defconfig
+++ b/configs/mvebu_armv8_le_defconfig
@@ -1,5 +1,11 @@
 BR2_aarch64=y
 BR2_TOOLCHAIN_EXTERNAL=y
+BR2_TOOLCHAIN_EXTERNAL_CUSTOM=y
+BR2_TOOLCHAIN_EXTERNAL_PATH="/path/to/compiler"
+BR2_TOOLCHAIN_EXTERNAL_CUSTOM_PREFIX="$(ARCH)-linux-gnu"
+BR2_TOOLCHAIN_EXTERNAL_HEADERS_3_17=y
+BR2_TOOLCHAIN_EXTERNAL_CUSTOM_GLIBC=y
+BR2_TOOLCHAIN_EXTERNAL_CXX=y
 BR2_TARGET_GENERIC_GETTY_PORT="ttyS0"
 BR2_ROOTFS_OVERLAY="user_files/"
 BR2_PACKAGE_BUSYBOX_SHOW_OTHERS=y
diff --git a/configs/mvebu_armv8_tiny_le_defconfig b/configs/mvebu_armv8_tiny_le_defconfig
index dd650d6..6c8227e 100644
--- a/configs/mvebu_armv8_tiny_le_defconfig
+++ b/configs/mvebu_armv8_tiny_le_defconfig
@@ -1,5 +1,11 @@
 BR2_aarch64=y
 BR2_TOOLCHAIN_EXTERNAL=y
+BR2_TOOLCHAIN_EXTERNAL_CUSTOM=y
+BR2_TOOLCHAIN_EXTERNAL_PATH="/path/to/compiler"
+BR2_TOOLCHAIN_EXTERNAL_CUSTOM_PREFIX="$(ARCH)-linux-gnu"
+BR2_TOOLCHAIN_EXTERNAL_HEADERS_3_17=y
+BR2_TOOLCHAIN_EXTERNAL_CUSTOM_GLIBC=y
+BR2_TOOLCHAIN_EXTERNAL_CXX=y
 BR2_TARGET_GENERIC_GETTY_PORT="ttyS0"
 BR2_ROOTFS_OVERLAY="user_files"
 BR2_ROOTFS_POST_IMAGE_SCRIPT="board/marvell/scripts/create_pxp_ramdisk.sh"
-- 
1.9.1

